docs/features/ssh.rst

```diff
@@ -41,32 +22,15 @@ Its full signature is::
                 password=None, key_filename=None, timeout=10):
         """Executes SSH command(s) on remote hostname."""
 
-Most of parameters can be read from ``robottelo.properties`` file.
+Most of parameters can be read from ``conf/server.yaml`` file.
 So in a properly configured project commands can be easily executed on host.
-The function returns a ``SSHCommandResult`` instance.
+The function returns a result object containing the attributes: stdout, stderr, and status.
 The following code was executed to generate the example of first section of
 this document::
 
     >>> from robottelo import ssh
     >>> print(ssh.command('cat /etc/redhat-release'))
-    SSHCommandResult(
-        stdout=['Red Hat Enterprise Linux Server release 7.2 (Maipo)', ''],
-        stderr='',
-        return_code=0,
-        output_format=None
-    )
-
-Another important function is ``execute_command``.
-It is similar to ``command``.
-But it reuses an existing connection.
-So several commands can be executed on a single connection, saving resources
-and execution time::
-
-    with ssh.get_connection(..) as connection:
-        ssh.execute_command('cp /orign /destiny', connection)
-        ssh.execute_command('chmod 0777 /destiny', connection)
-        ssh.execute_command("echo 'foo' > /destiny/bar")
-
+    Red Hat Enterprise Linux Server release 7.2 (Maipo)
```

Comment:
here the example should contains also the status and another example for stderr should be added, that will probably resolve the confusion that I had mentioned in other comments.